### PR TITLE
Implement --update-text option.

### PR DESCRIPTION
**Pull request by frknikiz**

### CODE REVIEW

File: src/main/java/fxlauncher/CreateManifest.java [hunk 1]
```diff
@@ -55,6 +56,10 @@ public static void main(String[] args) throws IOException, URISyntaxException {
                 if (named.containsKey("stop-on-update-errors"))
                     stopOnUpdateErrors = Boolean.valueOf(named.get("stop-on-update-errors"));
 
+                // Configure stopOnUpdateErrors
```
Comment (edvin) [inline src/main/java/fxlauncher/CreateManifest.java]: Can you fix this comment so it mentions updateText instead?

**Comment by edvin** [comment]
Great! I added a comment to line 59. I'll merge once you change the comment :)

**Comment by frknikiz** [comment]
I changed to comment :)

File: src/main/java/fxlauncher/CreateManifest.java (hunk 1 above)
Comment (frknikiz) [reply src/main/java/fxlauncher/CreateManifest.java]: I didn't realize when I was pasting :)

**Comment by edvin** [comment]
Perfect!

**Pull request merged**